ts/modules/networkgraph/networkgraph.src.ts

```diff
@@ -0,0 +1,1471 @@
+            draggable?: boolean;
+            link?: SVGAttributes;
+            marker?: NetworkgraphPointMarkerOptionsObject;
+            mass?: NodesSeriesOptions['mass'];
+            nodes?: Array<NetworkgraphPointOptions>;
+            states?: NetworkgraphSeriesStatesOptions;
+        }
+        interface NetworkgraphSeriesStatesInactiveOptions
+            extends LineSeriesStatesInactiveOptions
+        {
+            animation?: (boolean|AnimationOptionsObject);
+            linkOpacity?: number;
+        }
+        interface NetworkgraphSeriesStatesOptions
+            extends LineSeriesStatesOptions
+        {
+            inactive?: NetworkgraphSeriesStatesInactiveOptions;
+        }
+        class NetworkgraphPoint
+            extends LinePoint
+            implements DragNodesPoint, NodesPoint
+        {
+            public className: NodesPoint['className'];
+            public degree: number;
+            public fixedPosition: DragNodesPoint['fixedPosition'];
+            public formatPrefix: NodesPoint['formatPrefix'];
+            public from: NodesPoint['from'];
+            public fromNode: NetworkgraphPoint;
+            public getSum: NodesPoint['getSum'];
+            public hasShape: NodesPoint['hasShape'];
+            public isNode: NodesPoint['isNode'];
+            public linksFrom: Array<NetworkgraphPoint>;
+            public linksTo: Array<NetworkgraphPoint>;
+            public mass: NodesPoint['mass'];
+            public offset: NodesPoint['offset'];
+            public options: NetworkgraphPointOptions;
+            public series: NetworkgraphSeries;
+            public setNodeState: NodesMixin['setNodeState'];
+            public to: NodesPoint['to'];
+            public toNode: NetworkgraphPoint;
+            public destroy(): void;
+            public getDegree(): number;
+            public getLinkAttributes(): SVGAttributes;
+            public getLinkPath(): SVGPathArray;
+            public getMass(): Dictionary<number>
+            public init(
+                series: NetworkgraphSeries,
+                options: NetworkgraphPointOptions,
+                x?: number
+            ): Highcharts.NetworkgraphPoint;
+            public isValid(): boolean;
+            public redrawLink(): void;
+            public remove(redraw?: boolean, animation?: boolean): void;
+            public renderLink(): void;
+        }
+        class NetworkgraphSeries
+            extends LineSeries
+            implements DragNodesSeries, NodesSeries
+        {
+            public chart: NetworkgraphChart;
+            public createNode: NodesMixin['createNode'];
+            public data: Array<NetworkgraphPoint>;
+            public destroy: NodesMixin['destroy'];
+            public directTouch: boolean;
+            public drawTracker: TrackerMixin['drawTrackerPoint'];
+            public forces: Array<string>;
+            public hasDraggableNodes: boolean;
+            public isCartesian: boolean;
```

Comment:
Isn't this defined in `Series` class? The same for `noSharedTooltip`, `pointArrayMap` and `requireSorting`?

Reply:
Yes, but it gets here overwritten. So if you change the type in the parent, it will bring up a conflict, so that you have to update the overwrite here as well.

Reply:
Nice, thanks!